### PR TITLE
Add env var to frontend deployment for Jaeger

### PR DESCRIPTION
**Pull request by daxmc99**
Fixes https://github.com/sourcegraph/deploy-sourcegraph/pull/716

Sister [deploy-sourcegraph](https://github.com/sourcegraph/deploy-sourcegraph) change:https://github.com/sourcegraph/deploy-sourcegraph/pull/716

### CODE REVIEW

File: deploy-pgsql.sh [hunk 1]
```diff
@@ -15,7 +15,6 @@ docker run --detach \
     --name=pgsql \
     --network=sourcegraph \
     --restart=always \
-    --cpus=4 \
```
Comment (bobheadxi) [inline deploy-pgsql.sh:18]: why was this removed? :o

Comment (daxmc99) [reply deploy-pgsql.sh:18]: I was running the deploy scripts locally and kept seeing some CPU issues, this should not be here
